apps/next-evans/components/checkout/cart/Item.tsx

```diff
@@ -62,15 +62,15 @@ const Item: React.FC<Props> = ({
         </dd>
       </dl>
     </div>
-    <div className="w-1/3 md:w-2/5 flex text-center">
+    <div className="w-1/2 md:w-2/5 flex text-center">
       <ItemQuantity quantity={quantity} changeQuantity={changeQty} />
       <div className="w-1/2 md:w-1/3 flex flex-col justify-center price">
         <code className="px-1 font-sans text-gray-700 text-md md:text-lg antialiased md:tracking-wider">
           ${(price / 100).toFixed(2)}
         </code>
       </div>
       <div
-        className="hidden md:flex md:w-1/3 remove flex-col order-2 justify-center cursor-pointer"
+        className="flex w-1/3 remove flex-col justify-center cursor-pointer"
```

Comment:
this is not your fault, it was there before, but can you remove the `remove` utility? i don't think it's doing anything...